experimenter/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_ready_for_review_serializer.py

```diff
@@ -2466,7 +2466,7 @@ def test_not_localized_with_localizations(self):
                 experiment,
                 context={"user": self.user},
             ).data,
-            context={"user", self.user},
+            context={"user": self.user},
```

Comment:
Oh dang.  Annoying that `{a, b}` is valid syntax now.  Surprising nothing complained about not being able to find the user.... Oh you know why, cuz that's used to populate the user in the changelog generator and the review serializer doesn't save.

Reply:
I filed a bug about us refactoring this common pattern a while ago so we cant make this mistake again.